vocab/security/vocabulary.yml

```diff
@@ -215,51 +252,171 @@ class:
+      The aforementioned proofs are created to prove that some entity is delegating the authority to take some action to another entity. A verifier of the proof should expect the proof to express a `proofPurpose` of `capabilityDelegation` and reference a `verificationMethod` to verify it. The dereferenced `verificationMethod` MUST have a controller property that has a property of `capabilityDelegation` that references the `verificationMethod`. This indicates that the controller has authorized it for the expressed `proofPurpose`.
+
+  - id: capabilityInvocation
+    label: Capability Invocation Method
+    range: VerificationMethod
+    comment: |
+      A `capabilityInvocation` property is used to express that one or more `verificationMethods` are authorized to verify cryptographic proofs that were created for the purpose of invoking capabilities.
+      A `verificationMethod` MAY be referenced by its identifier (a URL) or expressed in full.
+      The aforementioned proofs are created to prove that some entity is attempting to exercise some authority they possess to take an action. A verifier of the proof should expect the proof to express a `proofPurpose` of `capabilityInvocation` and reference a `verificationMethod` to verify it. The dereferenced `verificationMethod` MUST have a controller property that, when dereferenced, has a property of `capabilityInvocation` that references the `verificationMethod.` This indicates that the controller has authorized it for the expressed `proofPurpose`.
+
+  - id: keyAgreement
+    label: Key agreement protocols
+    range: VerificationMethod
+    comment: Indicates that a proof is used for for key agreement protocols, such as Elliptic Curve Diffie Hellman key agreement used by popular encryption libraries.
+
+  - id: cryptosuite
+    label: Cryptographic suite
+    domain: sec:DataIntegrityProof
+    range: xsd:string
+    comment: A text-based identifier for a specific cryptographic suite.
+    see_also:
+      - label: vc-data-integrity
+        url: https://www.w3.org/TR/vc-data-integrity/#dataintegrityproof
+
+  - id: publicKeyMultibase
+    label: Public key multibase
+    domain: sec:VerificationMethod
+    range: xsd:string
+    comment: |
+      The public key multibase property is used to specify the multibase-encoded version of a public key. The contents of the property are defined by specifications such as ED25519-2020 and listed in the Linked Data Cryptosuite Registry. Most public key type definitions are expected to:
+      • Specify only a single encoding base per public key type as it reduces implementation burden and increases the chances of reaching broad interoperability.
+      • Specify a multicodec header on the encoded public key to aid encoding and decoding applications in confirming that they are serializing and deserializing an expected public key type.
+      • Use compressed binary formats to ensure efficient key sizes.
```

Comment:
These bullets (`<ul>`) may need some special treatment, to make multiple lines indent properly.

Reply:
*These* do need further attention, possibly to change the `•` to `*`, possibly to use whatever other markup makes the bullet-items "hang" properly.

Reply:
Let me take this separately, because this is an issue on the tool and not the vocabulary. I have raised a separate issue: https://github.com/w3c/yml2vocab/issues/8 and will resolve this particular conversation. It should not be a show-stopper for this particular vocabulary.